.changeset/six-donuts-march.md

```diff
@@ -0,0 +1,26 @@
+---
+"@spectrum-css/checkbox": major
+---
+
+# Checkbox S2 Migration
+
+Checkbox has some new control and color tokens.
```

Comment:
Could you add some elaboration here about the changes? Such as the removal of `.spectrum-Checkbox--sizeM` as medium is the default, high contrast updates, maybe font changes, and anything else that would be helpful to know about what has changed? I also see the thickness mod renamed, fixing its typo.